src/test/java/com/securian/creditcompass/OrderCalculatorTest.java

```diff
@@ -0,0 +1,69 @@
+package com.securian.creditcompass;
+
+import com.securian.creditcompass.entities.Claim;
+import com.securian.creditcompass.orderCalculator.OrderCalculator;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class OrderCalculatorTest {
+    private OrderCalculator orderCalculator;
+
+    @Test
+    void init(){
+        List<Claim> claims = createFakeClaims();
+        orderCalculator = new OrderCalculator(claims);
+        List<Claim> orderedClaims = orderFakeClaims(claims);
+        testOrder(orderedClaims);
+    }
+
+    //Creating fake claims for testing
+    List<Claim> createFakeClaims(){
+        List<Claim> claims = new ArrayList<>();
+        for (int i = 0; i <= 5; i++){
+            Claim claim = new Claim(i, "life", "Life",
+                    (float) 1000 + 1000 * i, 0, 0);
+            claim.setClaimStateTesting();
+            claims.add(claim);
+        }
+        for (int i = 0; i <= 2; i++){
+            Claim claim = new Claim(i, "life", "Illness",
+                    (float) 1000 + 1000 * i, 0, 0);
+            claim.setClaimStateTesting();
+            claims.add(claim);
+        }
+        Claim claim = new Claim(8, "illness",
+                "CompletedDocuments:N Illness useless text should be ignored", 131141.4F,
+                0, 0);
+        claim.setClaimStateTesting();
+        claims.add(claim);
+        return claims;
+    }
+
+    //Calculating the scores for the fake claims, as well as manually changing the urgency score for the others
+    //which has to be done since we can not and should not ever modify the date a claim was created.
+    List<Claim> orderFakeClaims(List<Claim> claims){
+        claims.get(claims.size() - 1).setUrgencyScore(40);
+        claims.get(claims.size() - 6).setUrgencyScore(50);
+        orderCalculator.calculateScores(claims);
+        List<Claim> orderedClaims = orderCalculator.orderClaims(claims);
+        return orderedClaims;
+    }
+
+    void testOrder(List<Claim> claims) {
```

Comment:
instead of calling this inside init(), this can be marked as the test method using @Test decorator (since this contains the assert statements). _Simply a testing convention improvement/refactoring suggestion. Doesn't need to be changed for this PR since method logic testing is covered._

Reply:
Thanks for the feedback